mantle/platform/qemu.go

```diff
@@ -255,6 +256,56 @@ func (inst *QemuInstance) WaitIgnitionError(ctx context.Context) (string, error)
 	return r.String(), nil
 }
 
+// CheckConsoleForBadness will only return if the instance
+// failed inside the CheckConsole.  The resulting string will
+// be a newline-delimited stream of JSON strings, as returned
+// by `journalctl -o json`.
+func (inst *QemuInstance) CheckConsoleForBadness(ctx context.Context) (string, error) {
```

Comment:
This looks like the almost the same code as the `WaitIgnitionError` function. We don't want to duplicate code, we should probably have a function that does deals with the journal processing that can be used in `WaitIgnitionError` and `CheckConsoleForBadness`.

Reply:
Also not 💯 % sure about this, but do we want to check the content of the journal or the serial console? AFAIU these are different.

Reply:
That's correct. For now I reverted changes to go back to the initial stages of the first PR related to the same issue. Once this compiles and works as expected I will request another review if you don't mind :)